lib/ClangImporter/ClangImporter.cpp

```diff
@@ -3262,6 +3262,64 @@ void ClangImporter::Implementation::lookupAllObjCMembers(
   }
 }
 
+bool ClangImporter::Implementation::loadNamedMembers(
+    const Decl *D, DeclName N, uint64_t contextData,
+    TinyPtrVector<ValueDecl *> &Members) {
+
+  auto *DC = dyn_cast<DeclContext>(D);
+  assert(DC && "loadNamedMembers on a Decl that's not a DeclContext");
+
+  auto *CD = D->getClangDecl();
+  assert(CD && "loadNamedMembers on a Decl without a clangDecl");
+
+  // There are 3 cases:
+  //
+  //  - The decl is from a bridging header, CMO is Some(nullptr)
+  //    which denotes the __ObjC Swift module and its associated
+  //    BridgingHeaderLookupTable.
+  //
+  //  - The decl is from a clang module, CMO is Some(M) for non-null
+  //    M and we can use the table for that module.
+  //
+  //  - The decl is from some other non-modular context, CMO is None.
```

Comment:
This can't really happen, because Clang can't distinguish between the bridging header and the non-modular context. It looks like the actual case in which this returns None is when `CD` is a forward declaration and there's no definition, but in that case you wouldn't ask for its members.

Reply:
fixed.